src/app/zap-templates/templates/chip/helper.js

```diff
@@ -248,6 +248,13 @@ function getAttributes(pkgId, options)
       // Enhanced the attribute with 'atomidId', 'discrete', chipType properties for convenience.
       att.atomicTypeId = atomic.atomicId;
       att.discrete     = atomic.discrete;
+
+      if (StringHelper.isString(att.type)) {
```

Comment:
What does that do for byte strings?

Reply:
Now, this is `chip::ByteSpan` for everything.